Source/swiftlint/Commands/RulesCommand.swift

```diff
@@ -7,10 +7,10 @@
 //
 
 import Commandant
-#if os(Linux)
-import Glibc
-#else
+#if !os(Linux)
```

Comment:
This change shouldn't be necessary. Could you please revert?

Reply:
Sure.  I was just trying to get rid of the ordered imports warning.

Reply:
SwiftLint 0.24.0 shouldn't have that warning.